controllers/nstemplateset/cluster_resources.go

```diff
@@ -237,6 +239,23 @@ CurrentObjects:
 					if err := r.setStatusUpdatingIfNotProvisioning(nsTmplSet); err != nil {
 						return false, err
 					}
+
+					// Special handling of DBaaSTenants is required because an admission webhook "vdbaastenant.kb.io" denies updates. Delete them and allow them to be created again as a workaround.
+					if currentObject.GetObjectKind().GroupVersionKind().Kind == "DBaaSTenant" {
```

Comment:
Do we know what kind of error we got for that? I saw the message, but I'm curious about the error type. Just thinking if we couldn't handle that case based on the error type rather than on GVK.

Reply:
We discussed in a separate thread: It's blocked by the DBaaS admission webhook. Not sure we would want to make it a generic behaviour. In the DBaaS case, they have mentioned it won't break functionality to delete and recreate the resource but that may not be the same for other resources and operators. So we'll move forward with a temp workaround if we cannot get a fix from DBaaS soon